### ISSUE TITLE
feature request: set rows directly

### ISSUE THREAD

**Issue by irgendwr** [rodaine/table#15]
First of: this looks like a very nice and simple replacement for [olekukonko/tablewriter](https://github.com/olekukonko/tablewriter) :)

What I'm missing though is an option to directly set the `rows [][]string`, since using `AddRow` wastes a lot of time rebuilding an array for no reason at all.

Example of my use-case:

I already have a `data [][]string` and want to set/add it to a table. I now have to loop over the `rowsOfStrings`, loop over each item again and create a **new array**, because `AddRow` takes `...interface{}` instead of `...string`, then `AddRow` loops over the array **again** just to convert the stings to strings and adding them to yet another newly created array and finally appending each of those new arrays to the `rows`.

```go
data := [][]string{
    []string{"1", "foo", "1337"},
    []string{"2", "bar", "1337"},
}

tbl := table.New("ID", "Name", "Amount")
for _, row := range data {
	newArr := make([]interface{}, len(row))
	for i, v := range row {
		newArr[i] = v
	}
	tbl.AddRow(newArr...) // <-- contains a for loop, conversion back to strings and yet another array
}
```

Instead I would prefer:

```go
data := [][]string{
    []string{"1", "foo", "1337"},
    []string{"2", "bar", "1337"},
}

tbl := table.New("ID", "Name", "Amount")
tbl.SetRows(data)
```

In [olekukonko/tablewriter](https://github.com/olekukonko/tablewriter) this is done quick and easy by using `AppendBulk` (although "set" would make more sense than append in my case) and I would be happy if this was possible with this lib as well.

TL;DR: I would be happy if there was a `func (t *table) SetRows(rows [][]string)`.
